en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/concepts/as-a-standalone-gateway.md

```diff
@@ -1,27 +1,27 @@
 # Choreo Connect as a Standalone Gateway
 
-Choreo Connect can be used as a standalone gateway without the API Manager as the control plane. APIs can be deployed via
+Choreo Connect can be used as a standalone Gateway without the API Manager as the control plane. APIs can be deployed via
 APICTL. APICTL can be used to create API projects by including the open API definitions. These API projects can be directly pushed to adapter. The adapter will deploy the API in the enforcer and the router.
 
-When use as a standalone gateway, Choreo Connect can be used with an external Secure Token Service(STS)
-which will provide signed JWTs, and Choreo Connect will validate those JWTs in the incoming requests. When used as a standalone gateway Choreo Connect
+When use as a standalone Gateway, Choreo Connect can be used with an external Secure Token Service(STS)
+which will provide signed JWTs, and Choreo Connect will validate those JWTs in the incoming requests. When used as a standalone Gateway Choreo Connect
 will not apply the subscription validation using the event hub as the control plane is not available. In order to achieve subscription
 validation we need to have subscription data also included in the incoming authentication JWT, which is called as a
 self-contained token.
```

Comment:
```suggestion
When Choreo Connect is used as a standalone Gateway,  
- You can connect it to an external Secure Token Service(STS) which will provide signed JWTs, and Choreo Connect will validate those JWTs in the incoming requests.      
- By default Choreo Connect *will not* apply subscription validations for access tokens since the control plane is not available. For access tokens, if the control plane is not connected but subscription validation is set to true, Choreo Connect expects subscription data also included in the incoming authentication JWT, which is called as a self-contained token.   
- by default Choreo Connect **will** apply the subscription validation for API keys. The structure of the API key JWT content is expected to be same as API keys generated by API Manager Publisher. Subscription validation for API keys will always expect Choreo Connect to have API Manager as the Control Plane since it validates against the most recent subscription information sent by the Control Plane.
- Enabling and disabling of subscription validations can be done by setting the parameter to true or false in [here](https://apim.docs.wso2.com/en/latest/deploy-and-publish/deploy-on-gateway/choreo-connect/configurations/enforcer-configurations/#token-service). Checkout `config.toml.template` located in `CHOREO-CONNECT_HOME/docker-compose/choreo-connect/conf` for examples.
 
```

Reply:
+1 for the changes.